src/ecmult_const_impl.h

```diff
@@ -12,6 +12,22 @@
 #include "ecmult_const.h"
 #include "ecmult_impl.h"
 
+/** Fill a table 'pre' with precomputed odd multiples of a.
+ *
+ *  The resulting point set is brought to a single constant Z denominator, stores the X and Y
+ *  coordinates as ge_storage points in pre, and stores the global Z in rz.
```

Comment:
```suggestion
 *  coordinates as ge_storage points in pre, and stores the global Z in globalz.
```

Who would have expected that the global Z is stored in `globalz`?